packages/api/src/utils/psa.js

```diff
@@ -132,11 +149,25 @@ export function transformAndValidate (payload) {
   // Validate CID.
   if (cid) {
     try {
-      normalizedCid = normalizeCid(cid)
-      opts.cid = cid
+      let c
+      try {
+        c = CID.parse(cid)
+        normalizedCid = c.toV1().toString()
+        opts.cid = cid
+      } catch (e) {
+        throw new PSAErrorInvalidData(INVALID_CID)
+      }
+
+      if (!SUPPORTED_CODECS.includes(c.code)) {
+        let message = PSACodecNotSupported.MESSAGE
+        if (c.code === 72) {
```

Comment:
Should `72` be documented somewhere to identify its meaning?

Reply:
I'll add it to a constant, to improve the readability.

Reply:
Good point, saved it in a constant and added a bit of explanation.